_dashboards/visbuilder.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: VisBuilder
+nav_order: 8
+---
+
+# VisBuilder
+
+VisBuilder is an experimental feature and shouldn't be used in a production environment. For updates on the feature's progress, or if you want to leave feedback that helps improve the feature, see the [git issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2280).
+{: .warning}
+
+You can use the VisBuilder visualization type in OpenSearch Dashboards to create data visualizations easily using a drag-and-drop gesture. With VisBuilder you have:
+
+* an immediate view of your data without the need to preselect the visualization output
+* the flexibility to change visualization types and index patterns quickly
+* the ability to easily navigate between multiple screens
+
+<img src="{{site.url}}{{site.baseurl}}/images/drag-drop-ui.png" alt="VisBuilder start screen">
+
+## Try out VisBuilder
+
+You can try out VisBuilder in your environment or in the [OpenSearch playground](https://playground.opensearch.org/app/home#/).
+
+### Try VisBuilder in the playground
+
+If you'd like to try out VisBuilder without installing OpenSearch locally, you can do so in the [Dashboards playground](https://playground.opensearch.org/app/wizard). The visualization type is enabled in the playground by default.
```

Comment:
same here

Reply:
https://playground.opensearch.org/app/vis-builder#/

Will be available after 2.4 release

Reply:
Revised to include the link